OpenRA.Game/Network/ReplayRecorder.cs

```diff
@@ -69,6 +69,9 @@ void StartSavingReplay(byte[] initialContent)
 
 		public void Receive(int clientID, byte[] data)
 		{
+			if (disposed)
+				return;
```

Comment:
So this is to prevent a crash but there isn't a way to check for disposed currently (other than making a public variable we check). Currently this is the only disposed object that crashes and that's because of the writer (once disposed accessing it causes an exception) and my understanding is that currently no other disposed object is checked because of this.